File: 7kyu/Arethenumbersinorder_v1.go

              // Arethenumbersinorder
              // https://www.codewars.com//kata/56b7f2f3f18876033f000307
        
              package kata

func InAscOrder(numbers []int) bool {
  if len(numbers) == 1 {
    return true
  } else {
    for i := 1; i < len(numbers); i++ {
      if numbers[i-1] > numbers[i] {
        return false
      }
    }
    return true
  }
}

            

File: 7kyu/Arethenumbersinorder_v1.js
// Arethenumbersinorder
// https://www.codewars.com//kata/56b7f2f3f18876033f000307

function inAscOrder(arr) {
  for (let i = 1; i < arr.length; i++) {
    if (arr[i] < arr[i - 1]) return false;
  }
  return true;
}
